### PR TITLE
Added unit tests for components in host and utility

### PR DESCRIPTION
**Pull request by vijayupadya**

### CODE REVIEW

File: DataProcessing/datax-host/pom.xml [hunk 1]
```diff
@@ -107,7 +107,7 @@ SOFTWARE
         <dependency>
             <groupId>org.scalatest</groupId>
             <artifactId>scalatest_2.11</artifactId>
-            <version>2.2.6</version>
+            <version>3.0.8</version>
```
Comment (carlbrochu) [inline DataProcessing/datax-host/pom.xml]: do we need to rev the versions of all componets to 1.3?

File: DataProcessing/datax-host/src/main/scala/datax/sink/OutputManager.scala [hunk 2]
```diff
@@ -72,6 +72,8 @@ object OutputManager {
     val onInitHandlers = sinkOperators.map(_._5).filter(_!=null)
     var shouldGeneratorProcessedSchema = processedSchemaPath!=null && !processedSchemaPath.isEmpty
 
+
```
Comment (carlbrochu) [inline DataProcessing/datax-host/src/main/scala/datax/sink/OutputManager.scala]: empty?

File: DataProcessing/datax-host/pom.xml (hunk 1 above)
Comment (carlbrochu) [reply DataProcessing/datax-host/pom.xml]: @rohit489

Comment (rohit489) [reply DataProcessing/datax-host/pom.xml]: We should upgrade the version of all the packages to 1.3.0-SNAPSHOT

**Comment by carlbrochu** [comment]
Will reopen when we get back to this

**Pull request closed without merging**